setup/config/webpack.config.dev.js

```diff
@@ -23,7 +23,7 @@ const config = {
     new ESLintPlugin({
       context: '../',
     }),
-    new webpack.IgnorePlugin({ resourceRegExp: /\.\/locale$/ }),
+    // new webpack.IgnorePlugin({ resourceRegExp: /\.\/locale$/ }),
```

Comment:
Hi @reyraa,
we had difficulty here .local file was ignored, I believe it had issues at the moment, but now it looks working fine do you know about it?